Project Writeup and Reflection.md

```diff
@@ -0,0 +1,136 @@
+
+One interesting thing to note: with TF-IDF, the compound score, or sentiment intensity, for each novel also has more variety. Frankenstein, in particular, has a compound score of -0.7184, showing that it is indeed uniquely depressing in its genre (top words include "miserable", "misery", "despair", "horror", "tears", and "grief", just like my life).
+
+The top 50 words, sentiments, and word clouds for each novel are included below.
+
+Also, going totally off topic: if you pickle the HTML text from https://google.com and open the .txt file, the encoded characters are a hodgepodge of questionable Chinese characters and radioactive symbols. If you copy it into Google translate and try to translate it into English, you end up with more repeating Chinese characters as well as gems such as:
+
+- "This item is eligible for Free International Shipping This item is eligible for Free International Shipping This item is eligible for Free International Shipping This item is eligible for Free International Shipping Important information about purchasing this product: This product is out of print and no longer available from the publisher Related promotions"
+- "The King of Cream is a kind of a cup of coffee and a cup of rice and a cup of rice."
+- "In this case, you will not be able to do so. If you have any questions, please do not hesitate to do so. If you have any questions, please do not hesitate to contact us."
+- "The United States of America" x100
+
+**Overall**
+
+Sentiment of Frankenstein:
+
+- {'neg': 0.139, 'neu': 0.7, 'pos': 0.162, 'compound': 1.0}
+
+Sentiment of Paradise_Lost:
+
+- {'neg': 0.12, 'neu': 0.724, 'pos': 0.157, 'compound': 1.0}
+
+Sentiment of The_Romance_of_Lust:
+
+- {'neg': 0.089, 'neu': 0.707, 'pos': 0.204, 'compound': 1.0}
+
+**Computed Using Word Frequency Analysis**
+
+Top 50 Words in Frankenstein:
+
+- ['man', 'life', 'father', 'shall', 'eyes', 'said', 'time', 'saw', 'night', 'elizabeth', 'mind', 'day', 'felt', 'death', 'heart', 'feelings', 'thought', 'dear', 'soon', 'friend', 'passed', 'miserable', 'heard', 'like', 'love', 'place', 'little', 'human', 'appeared', 'clerval', 'misery', 'friends', 'justine', 'country', 'nature', 'words', 'cottage', 'feel', 'great', 'old', 'away', 'hope', 'felix', 'return', 'happiness', 'know', 'despair', 'days', 'voice', 'long']
+
+Sentiment of Top 50 Words in Frankenstein:
+
+- {'neg': 0.169, 'neu': 0.492, 'pos': 0.339, 'compound': 0.9201}
+
+![Frankenstein WF Word Cloud](https://github.com/vivienyuwenchen/TextMining/blob/master/Frankenstein_wf.png)
+
+Top 50 Words in Paradise_Lost:
+
+- ['thir', 'thy', 'thou', 'thee', "heav'n", 'shall', 'th', 'god', 'earth', 'man', 'high', 'great', 'death', 'till', 'hath', 'hell', 'stood', 'day', 'good', 'like', 'things', 'night', 'light', 'farr', 'love', 'eve', 'o', 'world', 'adam', 'soon', 'let', 'hee', 'son', 'life', 'know', 'place', 'long', 'forth', 'self', 'mee', 'ye', 'way', 'power', 'hand', 'new', 'deep', 'end', 'fair', 'men', 'satan']
```

Comment:
Hmm. I wonder how much of the differences could be associated with unrecognized words - e. g., archaic spellings or contractions which the sentiment analyzer doesn't recognize and thus returns "neutral" for.